### ISSUE TITLE
Is there a way to customize the message set?

### ISSUE THREAD

**Issue by kamah82** [mtakaki/cachet-url-monitor#116]
Quick question. Is there a way to customize the error message generated? I.e. "The URL is unreachable: GET https://some.url".

Thanks.

**Comment by InFerYes** [comment]
You can change the code in configuration.py

https://github.com/mtakaki/cachet-url-monitor/blob/b339334b2c5e912403c0956b59e189e3629b5919/cachet_url_monitor/configuration.py#L170

**Comment by SmarLat** [comment]
I use the docker version of cachet-url-monitor, i can't edit configuration.py. It would be nice to customize the error message with the config.yml file!

**Comment by InFerYes** [comment]
Are you not able to access the contents of the container?

**Comment by SmarLat** [comment]
Yes i can edit with the docker exec -it, but still have the "The URL is unreachable: GET https://some.url". message

**Comment by JeremiePlam** [comment]
I also am running docker and I have edited both `/cachet_url_monitor/configuration.py` and `/build/lib/cachet_url_monitor/configuration.py` in the container to no avail.
It still displays a partial outage rather than a major outage both in the log and the Cachet status page.

Any help would be appreciated.

![image](https://github.com/mtakaki/cachet-url-monitor/assets/41689632/a1df48d7-0c76-4afa-99ac-1677dfcf6932)
